mbbsd/register.c

```diff
@@ -1078,12 +1078,78 @@ create_regform_request()
     return 1;
 }
 
+#define REGISTER_OK (0)
+#define REGISTER_ERR_INVALID_EMAIL (-1)
+#define REGISTER_ERR_EMAILDB (-2)
+#define REGISTER_ERR_TOO_MANY_ACCOUNTS (-3)
+#define REGISTER_ERR_CANCEL (-4)
+
+static int
+register_email_input(const char *userid, char *email)
+{
+    while (1) {
+	email[0] = 0;
+	getfield(15, "����{�ҥ�", REGNOTES_ROOT "email", "E-Mail Address", email, 50);
```

Comment:
For future reference, need to clean up hardcoded 50 length.

Reply:
Made a macro for this.